doc/source/cluster/configure-manage-dashboard.md

```diff
@@ -0,0 +1,323 @@
+
+Dashboard is included if you use `ray[default]`, `ray[air]`, or {ref}`other installation commands <installation>` and automatically started.
+
+To disable Dashboard, use the following arguments `--include-dashboard`.
+
+::::{tab-set}
+
+:::{tab-item} Single-node local cluster
+
+**Start the cluster explicitly with CLI** <br/>
+
+```bash
+ray start --include-dashboard=False
+```
+
+**Start the cluster implicitly with `ray.init`** <br/>
+
+```{testcode}
+:hide:
+
+import ray
+ray.shutdown()
+```
+
+```{testcode}
+import ray
+ray.init(include_dashboard=False)
+```
+
+:::
+
+:::{tab-item} VM Cluster Launcher
+Include the `ray start --head --include-dashboard=False` argument
+in the `head_start_ray_commands` section of the [Cluster Launcher's YAML file](https://github.com/ray-project/ray/blob/0574620d454952556fa1befc7694353d68c72049/python/ray/autoscaler/aws/example-full.yaml#L172).
+:::
+
+:::{tab-item} KubeRay
+
+```{admonition} Warning
+:class: warning
+It's not recommended to disable Dashboard because several KubeRay features like `RayJob` and `RayService` depend on it.
+```
+
+Set `spec.headGroupSpec.rayStartParams.include-dashboard` to `False`. Check out this [example YAML file](https://gist.github.com/kevin85421/0e6a8dd02c056704327d949b9ec96ef9).
+
+:::
+::::
+
+
+## Viewing built-in Dashboard API metrics
+
+Dashboard is powered by a server that serves both the UI code and the data about the cluster via API endpoints.
+Ray emits basic Prometheus metrics for each API endpoint:
+
+`ray_dashboard_api_requests_count_requests_total`: Collects the total count of requests. This is tagged by endpoint, method, and http_status.
+
+`ray_dashboard_api_requests_duration_seconds_bucket`: Collects the duration of requests. This is tagged by endpoint and method.
+
+For example, you can view the p95 duration of all requests with this query:
+
+```text
+
+histogram_quantile(0.95, sum(rate(ray_dashboard_api_requests_duration_seconds_bucket[5m])) by (le))
+```
+
+You can query these metrics from the Prometheus or Grafana UI. Find instructions on how to set these tools up {ref}`here <observability-visualization-setup>`.
+
+
+(observability-visualization-setup)=
+## Integrating with Prometheus and Grafana
+The optional integration with Prometheus and Grafana enables two features:
+- Access to default Grafana dashboard templates provided by Ray to visualize key system metrics for monitoring and debugging.
+- Embedding of Grafana visualizations in Ray Dashboard to view the time-series metrics.
+
+
+For the enhanced Ray Dashboard experience, like viewing time-series metrics, set up Prometheus and Grafana to integrate with Ray Dashboard.
+
+### Setting up Prometheus
+To render Grafana visualizations, you need Prometheus to scrape metrics from Ray clusters. Follow {ref}`the instructions <prometheus-setup>` to set up your Prometheus server and start to scrape system and application metrics from Ray Clusters.
+
+
+(grafana)=
+### Setting up Grafana
```

Comment:
I think this should be in metrics.rst like prometheus above?

Reply:
Currently, the metrics.rst is about "Collecting metrics". Grafana has nothing to do with this CUJ/user task. Grafana is needed for better Dashboard experience and we do have integration with it. That's why we need to mention it here.

Reply:
Let's keep it as is for now.